src/components/styles/Navbar.module.css

```diff
@@ -0,0 +1,87 @@
+.nav_container {
+  width: 100%;
+}
+
+nav {
+  background-color: #fff;
+  display: flex;
+  width: 100%;
+  gap: 2.5rem;
+  padding-left: 2.8rem;
+  align-items: center;
+}
```

Comment:
- [ ] As the previous reviewer requested, kindly align the nav links closer to the logo to match the given design: 
>yours: 
![image](https://github.com/Oscah01/the-book-store/assets/91486791/e2696876-c1e6-45d6-8aa6-61b57ade11a8)
>given:
![image](https://github.com/Oscah01/the-book-store/assets/91486791/1c13fe4a-d22f-4c4c-8279-397cc877e766)